.storybook/helpers/setup-storybook.helpers.ts

```diff
@@ -32,13 +34,31 @@ export function setupStorybook(
     );
 
   storyApi.add('Default', defaultStory, defaultParameters);
-
-  storyApi.add('Variants', variants.length ? combineStories(defaultStory, variants) : defaultStory, {
+  const combinedStories = variants.length ? combineStories(defaultStory, variants) : defaultStory;
+  const defaultParams = {
```

Comment:
I think this name is too similar to `defaultParameters`, so I would rename it `defaultVariantParameters`.